### PR TITLE
Refactor master

### PR DESCRIPTION
**Pull request by pheanex**
I did some refactoring.
If you want all of them i can also squash them together.

### CODE REVIEW

**Comment by alexeyklyukin** [comment]
👍

File: pg_view/__init__.py [hunk 1]
```diff
@@ -192,7 +189,7 @@ def main():
     clusters = []
 
     # now try to read the configuration file
-    config = (read_configuration(options.config_file) if options.config_file else None)
+    config = read_configuration(options.config_file if options.config_file else None)
```
Comment (a1exsh) [inline pg_view/__init__.py]: That doesn't seem right...

I think the paren should be closed earlier, before the `if`.

Comment (alexeyklyukin) [inline pg_view/__init__.py]: +1, although it's harmless in this case, as read_configuration emits None on None input.

**Comment by alexeyklyukin** [comment]
@pheanex could you, please, change line 192 (see comments above).  Then we merge.

File: pg_view/__init__.py (hunk 1 above)
Comment (pheanex) [inline pg_view/__init__.py]: Yes sure. Thank you for catching the mistake ... silly me :smile:

**Comment by alexeyklyukin** [comment]
👍

**Comment by a1exsh** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
@pheanex merged, thanks a lot!

**Pull request merged**